src/Microsoft.CodeAnalysis.Analyzers/Microsoft.CodeAnalysis.Analyzers.md

```diff
@@ -198,7 +198,7 @@ The 'helpLinkUri' value is used to show information when this diagnostic in the
 
 ## RS1016: Code fix providers should provide FixAll support
 
-A CodeFixProvider should provide FixAll support to enable users to fix multiple instances of the underlying diagnostic with a single code fix. See documenation at https://github.com/dotnet/roslyn/blob/master/docs/analyzers/FixAllProvider.md for further details.
+A CodeFixProvider should provide FixAll support to enable users to fix multiple instances of the underlying diagnostic with a single code fix. See documenation at <https://github.com/dotnet/roslyn/blob/master/docs/analyzers/FixAllProvider.md> for further details.
```

Comment:
Is it really the right fix? Isn't it better to do something like:

```suggestion
A CodeFixProvider should provide FixAll support to enable users to fix multiple instances of the underlying diagnostic with a single code fix. See [documenation](https://github.com/dotnet/roslyn/blob/master/docs/analyzers/FixAllProvider.md) for further details.
```

Reply:
Hyperlinking "documentation" is certainly better. The other fix was automated using `markdownlint -f` command.